lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -552,7 +549,7 @@ def create_network(self):
             if not self.check_mode:
                 resp = self.client.create_network(self.parameters.network_name, **params)
 
-                self.existing_network = self.client.inspect_network(resp['Id'])
+                self.existing_network = self.client.get_network(resp['Id'])
```

Comment:
Here we already know the full ID, so there's no need to first list all networks to find it again.